src/commands/interactive/detailViewTenant.ts

```diff
@@ -0,0 +1,108 @@
+import { Select } from "../../deps.ts";
+import { MeshTenant } from "../../mesh/mesh-tenant.model.ts";
+import { CmdGlobalOptions } from "../cmd-options.ts";
+import { MeshError } from "../../errors.ts";
+import { setupLogger } from "../../logger.ts";
+import { verifyCliAvailability } from "../../init.ts";
+import { loadConfig } from "../../config/config.model.ts";
+import { MeshAdapterFactory } from "../../mesh/mesh-adapter.factory.ts";
+
+export async function detailViewTenant(
```

Comment:
d:  at the moment the detail view comes when you select a tag and tenant after the EXPLORE TENANTS WITH MISSING TAGS selection.  unfortunately the tenants without tags are not shown. It would be great if we show the tenatns without tags and then enable the user to select one of them with the arrow keys or additionally by name.